package.json

```diff
@@ -51,59 +29,81 @@
     "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-jest": "^27.2.1",
     "eslint-plugin-react": "^7.21.0",
-    "file-loader": "^6.2.0",
     "hosted-git-info": "^2.7.1",
     "jest": "^29.5.0",
     "jest-environment-jsdom": "^29.5.0",
     "jest-transform-stub": "^2.0.0",
     "mockdate": "^3.0.5",
     "raf": "^3.4.1",
-    "redux-immutable-state-invariant": "^2.1.0",
     "regenerator-runtime": "^0.13.1",
-    "sass-loader": "^7.1.0",
-    "style-loader": "^0.23.1",
     "swagger-tools": "^0.10.4",
     "ts-jest": "29.1.0",
-    "ts-loader": "8.4.0",
     "typescript": "5.0.4",
     "webpack": "4.46.0",
     "webpack-bundle-analyzer": "^4.8.0",
     "webpack-cli": "3.3.12",
-    "webpack-dev-server": "3.11.3",
-    "yaml-loader": "^0.6.0"
+    "webpack-dev-server": "3.11.3"
   },
   "dependencies": {
+    "@babel/core": "7.21.8",
```

Comment:
So, we need all these deps in the "dependencies" rather than "devDependencies"?...

Reply:
Ops, I missed this comment. Yes, we need a number of deps to pre-compile assets.

```
$ RAILS_ENV=production NODE_ENV=production SECRET_KEY_BASE=rails/32947 bundle exec rake assets:precompile tmp:clear
....
EntryModuleNotFoundError: Entry module not found: Error: Can't resolve 'babel-loader' in '/opt/system'
    at /opt/system/node_modules/webpack/lib/Compilation.js:1075:31
    ...
```

Reply:
Aha, OK, thanks @akostadinov !

Reply:
Yeah, dev dependencies should be just linters and dev server. Everything required to make the app run (compilation, types, etc) has to be included as a dependency.

Reply:
With node 16. Idk why it worked with node 14.

Reply:
yeah, weird, right? :upside_down_face: